components/contract-tabs/phases/DropPhases.tsx

```diff
@@ -550,6 +556,10 @@ const DropPhasesForm: React.FC<DropPhases> = ({ contract, tokenId }) => {
                             )
                           }
                         />
+                        <FormHelperText>
+                          Even when set to unlimited, you&apos;ll still be
+                          limited by the block-limits of the blockchain.
```

Comment:
This is going to be confusing to more users than it helps. We either have to tell them what (roughly) the limit might be or explain it in a lot more detail or not try to explain it and hope for the best. (Current approach)